soc/xtensa/intel_s1000/memory.h

```diff
@@ -43,9 +43,16 @@
 #define MEM_ERROR_TEXT_SIZE			0x180
 #define MEM_ERROR_LIT_SIZE			0x8
 
-/* text and data share the same L2 HP SRAM on Intel S1000 */
-#define TEXT_BASE				(DT_L2_SRAM_BASE + L2_VECTOR_SIZE)
-#define TEXT_SIZE				0x16000
+/* text and data share the same L2 HP SRAM on Intel S1000.
+ * So, they lie next to each other.
+ */
+#define RAM_BASE			(DT_L2_SRAM_BASE + L2_VECTOR_SIZE)
+#define RAM_SIZE			0x400000
+
+/* Base address of the Interrupt Descriptor Table (IDT). This is a bogus
```

Comment:
this comment isn't correct. the IDT doesn't go here. it's a fake location for the intList section, which is used to construct the IDT.

Reply:
addressed.